packages/hdwallet/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Would it be better to name this something more general like @truffle/cryptography? Not something that should hold this PR back, just asking.

Reply:
Eh, I don't think so. This package will likely be removed from truffle and moved over to Ganache eventually (these methods were actually pulled from Ganache's source code, so it'd make sense for ganache to provide a package as a home for them).